lib/interfaces/src/InverterInterface.cpp

```diff
@@ -0,0 +1,185 @@
+#include <InverterInterface.h>
+
+/**
+ * Getters for the data
+ */
+
+InverterStatus_s InverterInterface::get_status() {
+    _feedback_data.status.new_data = false;
+    return _feedback_data.status;
+}
+
+InverterTemps_s InverterInterface::get_temps() {
+    _feedback_data.temps.new_data = false;
+    return _feedback_data.temps;
+}
+
+InverterPower_s InverterInterface::get_power() {
+    _feedback_data.power.new_data = false;
+    return _feedback_data.power;
+}
+
+MotorMechanics_s InverterInterface::get_motor_mechanics() {
+    _feedback_data.motor_mechanics.new_data = false;
+    return _feedback_data.motor_mechanics;
+}
+
+
+InverterControlFeedback_s InverterInterface::get_control_params() {
+    _feedback_data.control_feedback.new_data = false;
+    return _feedback_data.control_feedback;
+}
+
+
+/**
+ * receiving CAN messages
+ */
+
+void InverterInterface::receive_INV_STATUS(CAN_message_t &can_msg)
+{
+    // Unpack the message
+    INV1_STATUS_t unpacked_msg;
+    Unpack_INV1_STATUS_hytech(&unpacked_msg, can_msg.buf, can_msg.len);
```

Comment:
Why INV1? Does this only apply to one of the inverters, or is this message just reused?

It looks like it's being reused, but why named INV1?

Reply:
It's so that the inverter class works for all the inverters (they all have the same can frame data)